pkg/cmd/kafka/list/list.go

```diff
@@ -34,6 +34,7 @@ type kafkaRow struct {
 	Status        string `json:"status" header:"Status"`
 	CloudProvider string `json:"cloud_provider" header:"Cloud Provider"`
 	Region        string `json:"region" header:"Region"`
+	CustomerCloud string `json:"customer_cloud" header:"CustomerCloud"`
```

Comment:
Was this the agreed name?

Reply:
If not already agreed, let's revisit this by using the agreed header name.

Reply:
Also the header has to be `Customer Cloud` if we stick with that name:-)

Reply:
We have not officially agreed on a name for the output, @dimakis what do you think?

Reply:
There is no agreed names as of yet, I've brought this up a couple of times at meetings but so far we don't have names for `Red Hat OpenShift Cluster` or  `Your OpenShift Cluster`

Reply:
From those two I think `Red Hat OpenShift Cluster` sounds good to me

Reply:
Let's go with what we've hear and raise it in the sync call.